database/dynamodb.go

```diff
@@ -29,34 +21,50 @@ type AWSCredentials struct {
 var db *dynamodb.DynamoDB
 
 func init() {
+	defer func() {
+		if err := recover(); err != nil {
+			fmt.Println(err)
+		}
+	}()
 
-	env, found := os.LookupEnv(Constants["ENV"])
-	if !found {
-		// load the env values using the `setUpEnv` function.
-		utils.SetUpEnv()
+	env := os.Getenv(utils.ENV)
+	if env == utils.PROD {
+		log.Println(env, " is the env")
 	}
 
 }
 
 func getAWSCredentials() *AWSCredentials {
 
+	var found bool
 	awsCredentials := new(AWSCredentials)
-	awsCredentials.Region = os.Getenv(Constants["REGION"])
-	awsCredentials.AccessKey = os.Getenv(Constants["ACCES_KEY"])
-	awsCredentials.SecretKey = os.Getenv(Constants["SECRET_KEY"])
+
+	awsCredentials.Region, found = os.LookupEnv(utils.REGION)
+	if !found {
+		log.Panic("AWS Region not found ")
```

Comment:
instead of using log.Panic to handle missing environment variables, it would be more appropriate to return an error value and handle it at the calling site. Using log.Panic terminates the program immediately, which may not be desirable in all situations.

Reply:
Updated the code to use `log.Println`